src/test/ui/issues/issue-70724-add_type_neq_err_label-unwrap.stderr

```diff
@@ -33,7 +33,7 @@ LL |     assert_eq!(a, 0);
    = help: use parentheses to call the function: `a()`
    = note: required because of the requirements on the impl of `Debug` for `&fn() -> i32 {a}`
    = note: required by `std::fmt::Debug::fmt`
-   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
+   = note: this error originates in the macro `$crate::format_args` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
Is `$crate` meant to be exposed this way?

Reply:
Probably not. I'll open an issue to track resolving this.